docker-compose.yml

```diff
@@ -20,7 +20,9 @@ services:
   app:
     build:
       context: .
-      dockerfile: ./Dockerfile_dev
+      dockerfile: ./Dockerfile
+      target: development
+    platform: linux/amd64
```

Comment:
Mac users with an M1 chip will see an error when trying to build this image, so we've targeted a specific platform for now so that any devs wanting to use this container can do so (works on Windows / Mac)

Only downside is it slows down how long it takes to build the application image if you're machine's on different architecture